extensions/machine-learning/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "machine-learning",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
dont we need to update ADS depepndency version also?

Reply:
sure I'll do that

---
In reply to: [592654459](https://github.com/microsoft/azuredatastudio/pull/14659#discussion_r592654459) [](ancestors = 592654459)

Reply:
Do you need to though? You only need to update the version if there's new changes in core that you require for the extension to function. These fixes all seem to be in the extension so shouldn't require updating the engine version.

Reply:
yes but there were changes in core for some UI work, already in RC which this version of extension depends on it 

---
In reply to: [592664355](https://github.com/microsoft/azuredatastudio/pull/14659#discussion_r592664355) [](ancestors = 592664355)

Reply:
Cool, make sure you update the field in the gallery files when you update those as well